scopesim/tests/tests_effects/test_PSF.py

```diff
@@ -70,7 +70,7 @@ def test_returns_rotated_kernel_array_has_same_sum(self):
             plt.imshow(implane.data)
             plt.show()
 
-        assert np.sum(implane.data) == approx(1)
+        assert np.sum(implane.data) == approx(1, 1e-2)
```

Comment:
I've added this to dev_master as well as doing a pytest_mark-parameterize for 4 different angles (1,5, 15, 60) deg